rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_swift_stats.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Determine running version of swift
+  shell: "pgrep -fao swift | sed -r 's_.*(swift-[0-9.]+).*_\\1_'"
+  register: running_swift
+  delegate_to: "{{ groups['swift_hosts'][0] }}"
```

Comment:
@alextricity25 I wasn't sure about that, but looking over the maintenance plan they use `swift_proxy` so I'll change it to reflect what they are doing.